notebooks/rosa-demo-open-ai-wfaq.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<p>Shall we add a hyperlink to credential_example.env file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>